examples/customer_support_workflow/bank_api.py

```diff
@@ -0,0 +1,49 @@
+from flask import Flask, jsonify, request
```

Comment:
We don't use `Flask` in our examples. Please use `Fastapi` if you need API server. We already have fex examples with it and it exists in our examples pkgs group